### PR TITLE
Add SendForm skeleton.

### PR DESCRIPTION
**Pull request by adamjspooner**
This is my first attempt at a skeleton. It's missing the currency sections (the currency pane and the change link). I wanted to get the discussion started on best practices for organizing code.

@keybase/react-hackers, a penny for your thoughts.

### CODE REVIEW

**Comment by adamjspooner** [comment]
It also lacks the spinner overlay.

**Comment by adamjspooner** [comment]
@chrisnojima, this is ready for another review.

File: shared/wallets/send-form/body/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// @flow
+import * as React from 'react'
+import {Divider, Box2, Text} from '../../../common-adapters'
+import {styleSheetCreate} from '../../../styles'
+import AssetInput from '../asset-input'
```
[the remark this answers is not in the document]
Comment (chrisnojima) [reply shared/wallets/send-form/body/index.js]: ^ still needs to happen

File: shared/wallets/send-form/body/index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import Note from '../note/container'
+import Participants from '../participants/container'
+
+type Props = {
+  bannerInfo?: string,
+  isProcessing?: boolean,
+  onClick: Function,
+}
+
+const Spinner = () => <Box2 direction="vertical">Spinner</Box2>
```
Comment (chrisnojima) [inline shared/wallets/send-form/body/index.js]: i think we can make this spinner actually the correct implementation. just an overlay w/ the spinner in it over the correct components

Comment (adamjspooner) [reply shared/wallets/send-form/body/index.js]: Is the implementation in `teams/really-leave-team` a good example?

Comment (chrisnojima) [reply shared/wallets/send-form/body/index.js]: sure, it'll just be a box w/ a progressindicator inside (https://zpl.io/awrJJOg)
make sure its below the header

**Comment by adamjspooner** [comment]
I'm not sure if those `Spinner` updates are enough.

**Pull request merged**